component-library/src/app/pages/select-documentation/select-documentation.component.scss

```diff
@@ -0,0 +1,13 @@
+@use '../../../../../core-library/util/device' as device;
+@use '../../../../../core-library/tokens/template-const' as template-const;
+@use '../../../../../core-library/layout/layout-grid' as layout-grid;
+@use '../../../scss/page';
+@use '@ircc-ca/ds-sdc-core/typography/typography';
+
+@include page.template;
+
```

Comment:
Configurations main-component and sub-component sections should maintain 2 column layout in tablet and mobile view. Look at the button documentation for reference.